src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -95,6 +95,7 @@ public bool NeedsReanalysisOnOptionChanged(object sender, OptionChangedEventArgs
         {
             return e.Option.Feature == nameof(SimplificationOptions) ||
                    e.Option.Feature == nameof(CodeStyleOptions) ||
+                   e.Option.Feature == nameof(CodeStyleOptions2) ||
```

Comment:
❔ Why not use the name `CodeStyleOptions` for both?

Reply:
Yes, we do - in fact we have to for back compat. I will revert this change.